src/pages/Survey/SurveyBox/SurveyBox.js

```diff
@@ -0,0 +1,160 @@
+import React, { useState } from 'react';
+import { useHistory } from 'react-router-dom';
+
+// COMPONENTS
+import Radio from './Radio/Radio';
+import Select from './Select/Select';
+import Stepper from './Stepper/Stepper';
+import Loading from './Loading';
+
+// QUESTION DATA
+import {
+  SelectData,
+  questionOne,
+  questionTwo,
+  questionThree,
+} from '../SurveyData';
+
+// STYLES
+import * as S from './SurveyBoxEle';
+
+// APIKEY
+import { BASE_URL } from '../../../config';
+
+function SurveyBox() {
+  const [loading, setLoading] = useState(false);
+  const [selectedTown, setSelectedTown] = useState({
+    town: '',
+  });
+  const [radioValue, setRadioValue] = useState({
+    gender: '',
+    age: '',
+    career: '',
+  });
+  let [currentQ, setCurrentQ] = useState(0);
+  const history = useHistory();
+  // FUNCTIONS
+  const goToFindGosu = () => {
+    history.push('/findgosu');
+  };
+  const timeLoading = () => {
+    setTimeout(() => {
+      submitForm();
+    }, 5000);
+  };
+
+  const submitForm = () => {
+    const age = radioValue.age.slice(0, 2);
+    const career = radioValue.career.split('~')[0];
```

Comment:
```suggestion
    const [career] = radioValue.career.split('~');
```
개인적으로 인덱스 0 적어주는게 지저분해 보여서 구조분해 할당 해주면 좋은거 같아요

Reply:
오.. 저는 배열을 구조분해하는게 익숙치 않아서 좀 공부해봐야겠습니다 
저도 인덱스 0쓰기 넘 싫었어여...!!!!!!!!!